tools/SetupFlow/DevHome.SetupFlow/Models/ConfigureTargetTask.cs

```diff
@@ -516,11 +516,11 @@ public void UpdateHostConfig()
                 }
                 else
                 {
-                    GlobalLog.Logger?.ReportInfo($"HostConfig file contents are null or empty - HostConfigFileContents: {hostConfigContents}");
+                    _log.Information($"HostConfig file contents are null or empty - HostConfigFileContents: {hostConfigContents}");
                 }
             });
         }
     }
 
     private void OnThemeChanged(object sender, ElementTheme e) => UpdateHostConfig();
-}
+}
```

Comment:
It's another linting rule that files are supposed to end with a blank line, it got removed here.

Reply:
Yep.  Added.